cx_Freeze/hooks.py

```diff
@@ -77,6 +77,7 @@ def initialize(finder: ModuleFinder) -> None:
     finder.ExcludeModule("Queue")
     finder.ExcludeModule("rexec")
     finder.ExcludeModule("robotparser")
+    finder.ExcluedModule("sentry_sdk")
```

Comment:
Is this entry required? Usually, only load_sentry will be needed. Entries in this section are used to delete old modules like the ones that changed their name in the transition from py2 to py3.

Reply:
Probably not. I was trying to port across some code from PyInstaller to get the sentry_sdk working. Happy to go with those changes.